lib/server/imgix.js

```diff
@@ -18,9 +18,7 @@ export const imgixResolver = async (parent, { input }, context) => {
   const parentFSDocumentId = parent._id;
   if (!parentFSDocumentId) return null;
 
-  const fullParentFSFile = FSCollection.findOne({
-    _id: parentFSDocumentId,
-  });
+  const fullParentFSFile = await FSCollection.loader.load(parentFSDocumentId);
```

Comment:
sadly we need to do some extra work in the if clause beginning on line 32. For some reason the object returned by `loader.load` and the object returned by `FSCollection.findOne({` are different.

Honestly, a lot of the "file collection" stuff is a complete black box to me. I don't like it. And I don't question it.